kernel/build/config-amd64

```diff
@@ -7343,6 +7343,7 @@ CONFIG_TRACING=y
 CONFIG_GENERIC_TRACER=y
 CONFIG_TRACING_SUPPORT=y
 CONFIG_FTRACE=y
+CONFIG_TRACEFS_AUTOMOUNT_DEPRECATED=y
```

Comment:
drop this and update the mounts in talos

Reply:
we already mount `/sys/kernel/tracing`, so perhaps no change is needed